src/TasksModule/Components/TasksBoard/TasksBoard.tsx

```diff
@@ -0,0 +1,230 @@
+import React, { useContext, useEffect, useState } from "react";
+import styles from "./TasksBoard.module.css";
+import axios from "axios";
+import { baseUrl } from "../../../Constants/Components/Urls";
+import { motion } from "framer-motion";
+import { AuthContext } from "../../../Context/Components/AuthContext.js";
+
+
+export default function TasksBoard() {
+  const token = localStorage.getItem("userToken");
+  const {userRole}:string=useContext(AuthContext);
+  const [toDoTasks, setToDoTasks] = useState<object[]>();
+  // interface TaskData{
+  //   id:number,
+  //   newstatus:string,
+  //   oldstatus:string
+  // }
+  const [inProgressTasks, setInProgressTasks] = useState<object[]>();
+  const [doneTasks, setDoneTasks] = useState<object[]>();
+  const toDo: object[] = [];
+  const inProgress: object[] = [];
+  const done: object[] = [];
+  // const [isDraggingOver,setIsDraggingOver]=useState<boolean>(false);
+
+  const getUserTasks = async () => {
+    try {
+      const response = await axios.get(
+        `${baseUrl}/Task?pageSize=100&pageNumber=1`,
+        {
+          headers: { Authorization: token },
+        }
+      );
+      console.log("userTasks ", response.data.data);
+      setTasks(response.data.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const setTasks = (tasks: object[]) => {
```

Comment:
I don't like this name, as when first reading this function during code, it is tempting to think of it as a setter function of a variable called tasks, but this is not the case